plugins/@grouparoo/bigquery/src/initializers/plugin.ts

```diff
@@ -80,6 +80,14 @@ export class Plugins extends Initializer {
               description: "Private key of service account.",
               placeholder: "e.g. -----BEGIN PRIVATE KEY-----\nMII ...",
             },
+            {
+              key: "location",
+              displayName: "Location",
+              required: false,
+              description:
+                "Region or multi-region code of your dataset.  Required for datasets outside the US.",
+              placeholder: "e.g. EU",
+            },
```

Comment:
I would propose making this required, but having a default value: 

```suggestion
            {
              key: "location",
              displayName: "Location",
              required: false,
              description:
                "Region or multi-region code of your dataset",
              defaultValue: "US",
            },
````

This is how we handle things like the postgres port - https://github.com/grouparoo/grouparoo/blob/3e4bfda676026b34ec6a4285abb519db596f67ba/plugins/%40grouparoo/postgres/src/initializers/plugin.ts#L76-L80

Reply:
If we require location, having the default value will negate any migration issues, correct?

Reply:
I *think* so.  I would try it out to be sure